src/test/scala/dxWDL/compiler/NativeTest.scala

```diff
@@ -60,8 +60,8 @@ class NativeTest extends FlatSpec with Matchers with BeforeAndAfterAll {
                                       "/reorg_tests")
   override def beforeAll(): Unit = {
     // build the directory with the native applets
-    Utils.execCommand(s"dx mkdir -p ${TEST_PROJECT}:/${unitTestsPath}/applets/", quiet = true)
 
+    dxTestProject.newFolder(s"/${unitTestsPath}/applets/", parents = true)
```

Comment:
should we have logic in afterAll() to delete the folder?

Reply:
It is fine to just leave it there. It is a kind of lame optimization, so next time you run the test you don't have to re-create all the applets. It it nice to create the directory without shelling out to `dx` though. Nice touch :-)